docs/storybook-react/package.json

```diff
@@ -1,60 +1,15 @@
 {
   "name": "@metamask/storybook-react",
   "version": "0.0.0",
+  "private": true,
   "description": "storybook react",
-  "keywords": [
-    "MetaMask",
-    "Ethereum"
-  ],
-  "homepage": "https://github.com/MetaMask/metamask-design-system/tree/main/packages/storybook-react#readme",
-  "bugs": {
-    "url": "https://github.com/MetaMask/metamask-design-system/issues"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/MetaMask/metamask-design-system.git"
-  },
-  "license": "MIT",
-  "sideEffects": false,
-  "exports": {
-    ".": {
-      "import": {
-        "types": "./dist/index.d.mts",
-        "default": "./dist/index.mjs"
-      },
-      "require": {
-        "types": "./dist/index.d.cts",
-        "default": "./dist/index.cjs"
-      }
-    },
-    "./package.json": "./package.json"
-  },
-  "main": "./dist/index.cjs",
-  "types": "./dist/index.d.cts",
-  "files": [
-    "dist/"
-  ],
   "scripts": {
-    "build": "ts-bridge --project tsconfig.build.json --verbose --clean --no-references",
     "build-storybook": "storybook build",
-    "build:docs": "typedoc",
-    "changelog:update": "../../scripts/update-changelog.sh @metamask/storybook-react",
-    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/storybook-react",
-    "publish:preview": "yarn npm publish --tag preview",
-    "since-latest-release": "../../scripts/since-latest-release.sh",
-    "storybook": "storybook dev -p 6006",
-    "test": "NODE_OPTIONS=--experimental-vm-modules jest --reporters=jest-silent-reporter",
-    "test:clean": "NODE_OPTIONS=--experimental-vm-modules jest --clearCache",
-    "test:verbose": "NODE_OPTIONS=--experimental-vm-modules jest --verbose",
-    "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
-  },
-  "dependencies": {
-    "@metamask/design-tokens": "^4.0.0"
+    "storybook": "storybook dev -p 6006"
   },
   "devDependencies": {
     "@chromatic-com/storybook": "^1.9.0",
-    "@lavamoat/allow-scripts": "^3.2.1",
-    "@metamask/auto-changelog": "^3.4.4",
+    "@metamask/design-tokens": "^4.0.0",
```

Comment:
Removing as we use the dependencies in the root for allow-scripts and auto-changelog is not required here